app/Actions/Server/DeleteServer.php

```
@@ -2,16 +2,102 @@
+            ]);
+        }
+    }
+
+    private function deleteFromHetznerById(int $hetznerServerId, ?int $cloudProviderTokenId, int $teamId): void
+    {
+        try {
+            // Use the provided token, or fallback to first available team token
+            $token = null;
+
+            if ($cloudProviderTokenId) {
+                $token = CloudProviderToken::find($cloudProviderTokenId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\CloudProviderToken' in method 'deleteFromHetznerById'. (undefined)

(StaticAccess)